copyright

```diff
@@ -3787,6 +3787,12 @@ Copyright: None (placed in public domain by an anonymous author)
 License: CC0
 Comment: Uses public domain textures from texture.ninja. Meshes were made by an anonymous author in 2023-2024.
 
+Files:
+ images/ui/message?log?key*
+Copyright: TomGoodIdea (https://github.com/TomGoodIdea)
+License: CC-BY-SA-4.0
+Comment: Derived from works by Michael Zahniser (under the same license).
+
```

Comment:
How did you make the new graphic?

Reply:
iirc I took a similar sprite from the game files (outfitter key?) and modified it

Reply:
If that means you stretched it, I don't think that's enough to claim new copyright on it.

Reply:
It wasn't just a simple stretch (again, iirc), but I won't insist on keeping the copyright entry if you have concerns.

Reply:
So should I remove it or keep it?

Reply:
If we don't know what changes were made, then I'd prefer to either not have it, or leave the original copyright holder (MZ) and describe the changes you made. If the process of producing it was more involved, then it's probably fine as-is.

Reply:
~~if you'd asked me a few months ago, I'd remember the changes exactly~~
ok I'll remove it